src/components/MouseFollower/index.tsx

```diff
@@ -1,18 +1,26 @@
 import React from "react";
 
-import styles from "./style.css";
-
 interface MouseFollowerProps {
     pageX: number;
     pageY: number;
+    ref: React.RefObject<HTMLDivElement>;
 }
-const MouseFollower: React.SFC<MouseFollowerProps> = (props) => {
-    return (
-        <div
-            className={styles.follower}
-            style={{ left: `${props.pageX + 10}px`, top: `${props.pageY}px` }}
-        />
-    );
-};
+const MouseFollower = React.forwardRef<HTMLDivElement, React.PropsWithChildren<MouseFollowerProps>>(
+    (props, ref) => {
```

Comment:
Enabled ref forwarding so the thumbnail `Popover` can use this `div` as a container. This allows it to move together with the MouseFollower's `div`.